test/hls/hls_parser_unit.js

```diff
@@ -2587,6 +2587,106 @@ describe('HlsParser', () => {
     expect(actual).toEqual(manifest);
   });
 
+  describe('When config.hls.disbaleCodecGuessing is set to true', () => {
```

Comment:
Nit: `disbaleCodecGuessing` -> `disableCodecGuessing`

Reply:
Thank you, done ✅